config/config.go

```diff
@@ -774,6 +775,9 @@ func (cfg *MempoolConfig) ValidateBasic() error {
 	if cfg.MaxTxBytes < 0 {
 		return errors.New("max_tx_bytes can't be negative")
 	}
+	if cfg.CacheSize < cfg.Size {
```

Comment:
I am not sure that this must be a requirement. If someone doesn't want any caching?

Reply:
I think the mempool is not prepared to not have caching as it doesn't behave as expected when `cache_size` is set to 0. 

I suspect that the implementation of `LRUTxCache` is incorrect. For example, while the capacity of the cache (the `size`) is set to 0 in the constructor and it never changes, the internal data structures `cacheMap` and `list` will grow when we push a tx to the cache.

Reply:
In this case, this method should reject size 0. In any case, we should provide to users a better explanation regarding the actual use of the cache.